### PR TITLE
IP2 - 10

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: unabridged-works-volume-2/original/10-concerning-persecution.adoc [hunk 1]
```diff
@@ -918,11 +1001,13 @@ but a naked, upright waiting on him for instruction,
 who can only guide the spirit of man aright, governments would not prove so difficult,
 nor the success therein so dangerous.
 
-=== A BRIEF ACCOUNT OF THAT STIFFNESS, RESOLVEDNESS AND SUPPOSED STUBBORNNESS, WHICH BY MANY IS OBJECTED AGAINST THE PEOPLE CALLED QUAKERS
+====
+
+=== A Brief Account of That Stiffness, Resolvedness and Supoorted Stubbornness, Which by Many Is Objected Against the People Called Quakers
```
Comment (jaredh159) [inline unabridged-works-volume-2/original/10-concerning-persecution.adoc:1006]: Whoops, you accidentally changed `SUPPOSED` into `Supoorted`.  

While you're fixing that, would you mind making this an [.old-style]?  with a `/` before `Which`, like this:

```ASCIIDOC
=== A Brief Account of That Stiffness, Resolvedness and Supposed Stubbornness, / Which by Many Is Objected Against the People Called Quakers
```

File: unabridged-works-volume-2/original/10-concerning-persecution.adoc [hunk 2]
```diff
@@ -252,11 +267,15 @@ in relation to any thing that they do from this principle, is persecution with t
 or lifting up of the power, either of a particular person, or of a magistrate,
 against such.
 
-==== 4. THE GROUNDS OF PERSECUTION, OR WHAT ARE THE THINGS THAT CAUSE THE ONE SPIRIT AND PRINCIPLE TO PERSECUTE, AND THE OTHER TO BE PERSECUTED
+==== 4. The Grounds of Persecution, or What Are the Things That Cause the One Spirit and Priciple to Persecute, and the Other to Be Persecuted
```
Comment (jaredh159) [inline unabridged-works-volume-2/original/10-concerning-persecution.adoc:270]: Whoops... `PRINCIPLE` got changed to `Priciple`

File: unabridged-works-volume-2/original/10-concerning-persecution.adoc [hunk 3]
```diff
@@ -377,14 +405,21 @@ but cherish and nurse it up as much as in them lies,
...
 3+++.+++ A third way of persecution is by the hand of violence, without either law,
 or so much as pretence to law.
 Thus the persecuting spirit, when it hath power in its hand, and is out of fear,
 smiteth (with the open fist of wickedness) that which is an enemy to,
 and stands a witness against, its wickedness.
 
-==== 6. THE ENDS OF PERSECUTION, OR WHAT THE PERSECUTING SPIRIT AIMS AT IN ITS PERSECUTING, AND WOULD FAIN ATTAIN THEREBY
+====
+
+==== 6. The End of Persection, or What the Persecuting Spirit Aims At in Its Persecuting, and Would Fain Attain Thereby
```
Comment (jaredh159) [inline unabridged-works-volume-2/original/10-concerning-persecution.adoc:417]: `ENDS` accidentally got changed to `End`

File: unabridged-works-volume-2/original/10-concerning-persecution.adoc [hunk 4]
```diff
@@ -863,6 +937,11 @@ under a pretence of conscience.
 
 [.discourse-part]
 Answer.
+
+[.numbered-group]
+====
+
+[.numbered]
```
Comment (jaredh159) [inline unabridged-works-volume-2/original/10-concerning-persecution.adoc:944]: This is another case like in the last PR, where `Answer.` gets put in its own paragraph by the new formatting, which we don't want.  Let's do the same here, remove the  DP on line 938 and make this just a .numbered, like this:

```ASCIIDOC
[.numbered-group]
====

[.numbered]
_Answer 1._ Conscience is of God;
```

File: unabridged-works-volume-2/original/10-concerning-persecution.adoc (hunk 1 above)
Comment (maijapapaya) [reply unabridged-works-volume-2/original/10-concerning-persecution.adoc:1006]: idk how I do all these typos! I feel like I really pay attention! 😞

**Comment by jaredh159** [comment]
Good deal, thanks, merging. 👍

**Pull request merged**